source/Grid/Grid.js

```diff
@@ -63,6 +64,12 @@ export default class Grid extends Component {
     noContentRenderer: PropTypes.func.isRequired,
 
     /**
+     * Optional callback invoked on mouse-wheel activity.
+     * (event: SyntheticEvent): void
+     */
+    onWheel: PropTypes.func,
```

Comment:
Let's remove it for now. Baby steps with this sort of thing. I've been experimenting _a lot_ with regard to scrolling performance. This is another combination to add to my list of things to play with.